src/DynamoCore/DynamoCore.csproj

```diff
@@ -132,6 +135,9 @@
     <Copy SourceFiles="$(SolutionDir)..\doc\distrib\InstrumentationConsent.rtf" DestinationFolder="$(OutputPath)" />
     <Copy SourceFiles="$(SolutionDir)..\doc\distrib\ADPAnalyticsConsent.rtf" DestinationFolder="$(OutputPath)" />
     <Copy SourceFiles="@(NodeHelpFiles)" DestinationFolder="$(OutputPath)\en-US\fallback_docs\" />
+    <Copy SourceFiles="@(NodeHelpJpgImageFiles)" DestinationFolder="$(OutputPath)\fallback_docs_images\" />
+	<Copy SourceFiles="@(NodeHelpPngImageFiles)" DestinationFolder="$(OutputPath)\fallback_docs_images\" />
+	<Copy SourceFiles="@(NodeHelpGifImageFiles)" DestinationFolder="$(OutputPath)\fallback_docs_images\" />
```

Comment:
with `<Copy SourceFiles="@(NodeHelpFiles)" DestinationFolder="$(OutputPath)\en-US\fallback_docs\" />` still exist, do we double copy the images?

Reply:
Yes, we will have the images duplicated only in Debug\en-US\fallback_docs and Debug\fallback_docs_images, otherwise I will need to find a way to one of the next approaches: 

- copy all the files except the images (jpg, png and gif) into the Debug\en-US\fallback_docs folder.
- copy all the files into the Debug\en-US\fallback_docs folder. and then delete just the images
- copy just some kind of files (like md, dyn files) into the Debug\en-US\fallback_docs folder.

Please let me know what you think. Thanks

Reply:
Please continue to explore, we would prefer no duplicate of files. On a side note, can we share dyns as well? I guess not since they are localized as well?

Reply:
I've commited a fix so we won't have duplicate image files, see commit: https://github.com/DynamoDS/Dynamo/pull/14429/commits/e26218cbb32eabf94e93226fb7eb0963ee360f5b

About dyn files, they can be also be shared the only problem is that we will need to change the insert process so we take the dyn file from the shared path. 
In this case we will have only the md files in the fallback_docs folder and I'm not sure if they can be shared or not but also I've found that in the fallback_docs folder we have one .SAT file and 3 .TXT files.
Thanks

Reply:
Let's leave the .SAT and .TXT files. I have been thinking about DYNs. I looked at a few examples from zh-CN and they remain in English (not localized). I will discuss with @Amoursol If we want to share them as well ot further reduce the file numbers

Reply:
If localization team can confirm DYNs for node help docs are not localized, per https://autodesk.slack.com/archives/C01FVCEMYTA/p1695325397480959. We can move DYNs to shared folder as well. Let's leave this PR open for another day. The current change looks good

Reply:
hi @RobertGlobant20 After discussion, I would like to have DYNs also shared, would you update our approach with that?